source/assets/sass/config/color/_content.scss

```diff
@@ -31,3 +31,7 @@ $sl-color--info-lighter: color.adjust(brand.$sl-color--patina, $lightness: 47%);
 $sl-color--error: #cf0254;
 $sl-color--warn: #c14e00;
 $sl-color--success: #168073;
+
+// Darker Shades of existing colors for use on a light gray background
+$sl-color--text-medium-dark: #5e6470; // darker version of pale-sky
+$sl-color--action-dark: #a12d69; // darker version of action
```

Comment:
Should these be defined in terms of `color.scale()` or `color.adjust()`?

Reply:
@nex3 I can update that. I just quickly used a color app's down arrows until it passed. I can switch to a more readable format and look up the exact adjustments to use to make it work based on the brand colors.

Reply:
I'd appreciate it! We may also move to new color spaces for color conversions at some point, at which point knowing the underlying logic is valuable because we can just make the results more accurate.

Reply:
Added in https://github.com/sass/sass-site/pull/1177/commits/93a4f1a6c70e1daa194be2ba97c98374716ccdf9